src/containers/auth/resetPassword/styles.module.scss

```diff
@@ -45,11 +46,11 @@
     width: 20%;
   }
 
-  .btn_right_position {
+  .btnRightPosition {
     align-self: flex-end;
   }
 
-  .btn_margin_top {
+  .btnMarginTop {
     margin-top: 25px;
   }
 
```

Comment:
Can we unite this in one style and rename it? i mean not btn

Reply:
Done